### PR TITLE
Makes teslas explode when touching another tesla

### PR DESCRIPTION
**Pull request by ChesterTheCheesy**
# General Documentation
### Intent of your Pull Request
I made an alternative here: [#11191]
Theos was mad because people stacked way too many teslas in one containment, making lag. This PR prevents stacking Teslas that are in the same containment destroying both of them and creating a massive explosion.

### Why is this change good for the game?
Prevents tesla stacking, which causes lag

# Wiki Documentation
should be noted in bold red NOT TO STACK THOSE TESLAS OR THEY WILL FUCKING SUPERCOLLIDE

### Briefly describe your PR and the impacts of it, in layman's terms. 
Teslas now destroy each other on contact with another teslas in close proximity, causing a massive explosion and making an admin log.

### What should players be aware of when it comes to the changes your PR is implementing?
To not stack teslas under one containment, make a cage for each instead.

### What general grouping does this PR fall under? 
Engineering tweak

# Changelog
:cl:  
tweak: teslas now explode when touching each other, deleting ~one~ both of them
/:cl:

### CODE REVIEW

**Comment by alexkar598** [comment]
Not sure im too much a fan of fucking up rounds because you missed a pr

**Comment by Xoxeyos** [comment]
Why not just have it so the more energetic tesla ball siphons the other tesla ball into it's current miniball stack, perhaps allowing the max cap of the miniballs to go up?

**Comment by ChesterTheCheesy** [comment]
> 
> 
> Not sure im too much a fan of fucking up rounds because you missed a pr

Stacking teslas is so uncommon that people that do it either look at wiki (which as i pointed, should be updated to accomodate this change) or are regular with server updates.

**Comment by ChesterTheCheesy** [comment]
> 
> 
> Why not just have it so the more energetic tesla ball siphons the other tesla ball into it's current miniball stack, perhaps allowing the max cap of the miniballs to go up?

I guess that's a cool idea, but we also don't want people to stack like 10 of them, because each miniball is still a tesla at it's core.
What do you say that the tesla becomes "Supercharged", slightly redder, with maximum of 15 miniballs, but explodes on contact with any more tesla? Or it can just stack those "upgrades", but get 1 more zapping range and like, 2-3 more max miniballs per tesla consumed.

**Comment by ToasterBiome** [comment]
> > Why not just have it so the more energetic tesla ball siphons the other tesla ball into it's current miniball stack, perhaps allowing the max cap of the miniballs to go up?
> 
> I guess that's a cool idea, but we also don't want people to stack like 10 of them, because each miniball is still a tesla at it's core.
> What do you say that the tesla becomes "Supercharged", slightly redder, with maximum of 15 miniballs, but explodes on contact with any more tesla? Or it can just stack those "upgrades", but get 1 more zapping range and like, 2-3 more max miniballs per tesla consumed.

I love this idea and should definitely be done instead of cucking tesla stacking, maybe a cool blue one or something would be neat

**Comment by Marmio64** [comment]
wouldnt the explosion destroy the containment and leave one tesla alive, making tesloose? If it does do that, maybe make the collision destroy both teslas so that the entire round isnt completely fucked from someone making a little mistake.

**Comment by ChesterTheCheesy** [comment]
For now i'll turn this PR into explosion and destruction of both teslas, but i'll work on teslas stacking idea.

**Comment by Readystorm546** [comment]
I like the other PR much more.
